tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -873,3 +874,83 @@ def crop_center(img, cropx, cropy, cropz):
         endz = min(startz + cropz, z)
 
         return img[startz:endz, starty:endy, startx:endx, :]
+
+
+class AffineTransform(Preprocessor):
```

Comment:
This does not look like generic affine transform. Suggest to change the name to specific one. Not sure what to suggest, but it is some kind of crop.